.env

```diff
@@ -0,0 +1 @@
+SECRET_KEY = 'django-insecure-=8!l&n-b&q+^xn7df+(!$w_!$^e2wzz^)ji1e#l3_i%m(n72bd'
```

Comment:
you shouldn't push this

Reply:
add  gitignore so all these unnecessary files like .idea and pycache don't get committed